Libraries/Template/Components/LinkList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { View, Text, StyleSheet, TouchableOpacity, Linking } from 'react-native'
+
+const links = [
+  {
+    title: 'The Basics',
+    link: 'https://facebook.github.io/react-native/docs/tutorial',
+    description:
+      'Read the docs on what do do once seeing how to work in React Native.',
+  },
+  {
+    title: 'Style',
+    link: 'https://facebook.github.io/react-native/docs/style',
+    description: 'All of the core components accept a prop named style.',
+  },
+  {
+    title: 'Layout',
+    link: 'https://facebook.github.io/react-native/docs/flexbox',
+    description:
+      'A component can specify the layout of its children using the flexbox specification.',
+  },
+  {
+    title: 'Components',
+    link: 'https://facebook.github.io/react-native/docs/components-and-apis',
+    description: 'The full list of components and APIs inside React Native.',
+  },
+  {
+    title: 'Navigation',
+    link: 'https://facebook.github.io/react-native/docs/navigation',
+    description:
+      'How to handle moving between screens inside your application.',
+  },
+  {
+    title: 'Networking',
+    link: 'https://facebook.github.io/react-native/docs/network',
+    description: 'How to use the Fetch API in React Native.',
+  },
+  {
+    title: 'Help',
+    link: 'https://facebook.github.io/react-native/help',
+    description:
+      'Need more help? There are many other React Native developers who may have the answer.',
+  },
+]
+
+const LinkList = () => (
+  <View style={styles.container}>
+    {links.map((item, index) => {
+      return (
+        <View key={index}>
+          <View style={styles.separator} />
+          <TouchableOpacity
+            onPress={() => Linking.openURL(item.link)}
+            style={styles.linkContainer}
+          >
+            <Text style={styles.link}>{item.title}</Text>
+            <Text style={styles.description}>{item.description}</Text>
+          </TouchableOpacity>
+        </View>
+      )
+    })}
+  </View>
+)
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 32,
+    paddingHorizontal: 24,
+  },
+  linkContainer: {
+    flexWrap: 'wrap',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: 8,
+  },
+  link: {
+    width: '40%',
```

Comment:
Hm, I wonder how this will look on iPad and other tablets? What do you think about making this a `maxWidth` instead, given that we know the titles will generally be relatively short.

Reply:
This was the result with `maxWidth`:
<img width="635" alt="Screen Shot 2019-05-02 at 9 33 47 PM" src="https://user-images.githubusercontent.com/15311858/57114919-28108e00-6d22-11e9-897b-8c62dc93bbd3.png">

So I remake it with the `Flex` property. This was the result:
<img width="635" alt="Screen Shot 2019-05-02 at 9 35 51 PM" src="https://user-images.githubusercontent.com/15311858/57114942-4aa2a700-6d22-11e9-88ea-2af4e61325b3.png">